corehq/apps/data_interfaces/interfaces.py

```diff
@@ -141,6 +145,43 @@ def bulk_response(self):
         )
 
 
+@location_safe
+class CaseCopyInterface(CaseReassignmentInterface):
+    name = gettext_noop("Copy Cases")
+    slug = "copy_cases"
+    report_template_path = 'data_interfaces/interfaces/case_management.html'
+    action = "copy"
+    action_text = gettext_lazy("Copy")
+
+    @property
+    @memoized
+    def es_results(self):
+        query = self._build_query()
+        owner_id = self.request.GET.get('individual')
+        if owner_id:
+            query = query.owner(owner_id)
+
+        return query.run().raw
+
+    @property
+    def template_context(self):
+        context = super(CaseReassignmentInterface, self).template_context
+        context.update({
+            "action": self.action,
+            "action_text": self.action_text,
+        })
+        return context
+
+    @property
+    def fields(self):
+        return [
+            'corehq.apps.reports.filters.users.SelectMobileWorkerFilter',
+            'corehq.apps.reports.filters.select.MultiCaseTypeFilter',
+            'corehq.apps.reports.standard.cases.filters.CaseSearchFilter',
+            'corehq.apps.reports.standard.cases.filters.SensitiveCaseProperties',
```

Comment:
nit: You could use `super.fields + ['corehq.apps.reports.standard.cases.filters.SensitiveCaseProperties']` to be DRY

Reply:
Ah, right. This is nice